src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,73 @@
+package duke.ui;
+
+import duke.task.TaskList;
+import duke.task.Task;
+import java.util.Scanner;
+
+public class Ui {
+    private static final String LINE = "    ____________________________________________________________";
+    private static final String INDENTATION = "     ";
+    private Scanner sc = new Scanner(System.in);
+
+    public String readCommand() {
+        return sc.nextLine();
+    }
+
+    public void printMessage(String message) {
+        System.out.println(LINE + "\n" + message + "\n" + LINE + "\n");
+    }
+
+    public String makeIndent(String message) {
+        return INDENTATION + message;
+    }
+
+    public void displayGreeting() {
+        printMessage(makeIndent("Hi, how are you doing?! I'm JRH2000\n") +
+                makeIndent("How can I help you?\n"));
```

Comment:
Perhaps the `+` operator can be placed on the second line instead to improve readability.